features/step_definitions/basic_steps.rb

```diff
@@ -19,3 +19,12 @@
   Then("I click {string} button") do |login|
     click_button login
   end
+
+  Given(/^I am logged in as "([^"]*)"$/) do |name|
```

Comment:
There is no need for Regex, please refactor this step.

Reply:
OK. Refactored without the RegExp.